utils/ttyd/files/ttyd.init

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=50
+USE_PROCD=1
+
+NAME=ttyd
+PROG=/usr/bin/$NAME
+
+validate_section_ttyd()
+{
+	uci_load_validate ttyd ttyd "$1" "$2" \
+		'enable:bool:1' \
+		'port:port' \
+		'interface:string' \
+		'credential:string' \
+		'uid:uinteger' \
+		'gid:uinteger' \
+		'signal:uinteger' \
+		'url_arg:bool' \
+		'readonly:bool' \
+		'client_option:list(string)' \
+		'terminal_type:string' \
+		'check_origin:bool' \
+		'max_clients:uinteger' \
+		'once:bool' \
+		'index:string' \
+		'ipv6:bool' \
+		'ssl:bool' \
+		'ssl_cert:file' \
+		'ssl_key:file' \
+		'ssl_ca:file' \
+		'debug:uinteger' \
+		'command:string'
+}
+
+ttyd_instance()
+{
+	[ "$2" = 0 ] || {
+		echo "validation failed"
+		return 1
+	}
+
+	[ "$enable" = 0 ] && return 1
+	[ -z "$command" ] && return 1
+
+	[ "${interface::1}" = @ ] && {
```

Comment:
Will your luci app write interface value with the `@` prefix, or it's used by the default value only?

Reply:
I think it makes more sense to prevent the selection of alias interfaces in the LuCI app. Since this daemon does not appear to bind to ip addresses, being able to select an alias interfaces makes not much sense.

Reply:
@jow- both interface and ip address are supported by ttyd's interface option.

Reply:
Ah I see, but in this case the `@alias` interface should be resolved to an IP address so that the daemon binds to this specific IP.

Reply:
@jow- Because ttyd's interface argument takes a Linux interface name, not netifd interface. I think using `@lan` is better than hard-coded `br-lan`.